pyiron_atomistics/vasp/base.py

```diff
@@ -1631,7 +1630,9 @@ def restart_from_charge_density(
         new_ham = self.restart(job_name=job_name, job_type=job_type)
 
         if new_ham.__name__ == self.__name__:
-            new_ham.restart_file_list.append(self.get_workdir_file("CHGCAR"))
+            if "CHGCAR" not in self.files.list():
+                raise FileNotFoundError("The CHGCAR has to be available in the working directory: " + self.working_directory)
```

Comment:
While these separate checks are a bit annoying, if we do not do this it raises either a `KeyError` or an `AttributeError` both is not exactly what we want. We could think about changing this in `pyiron_base`.